apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/SwapConnectWidget/index.tsx

```diff
@@ -1,105 +0,0 @@
-import * as remote from "@electron/remote";
```

Comment:
if you remove the `SwapConnectWidget`, then the `swapConnectWebviewPreloader` related files should be removed as well:
- `apps/ledger-live-desktop/src/swapConnectWebviewPreloader/index.ts`
- `apps/ledger-live-desktop/tools/config/swapConnectWebviewPreloader.esbuild.js`

Reply:
Yep got it noted 👍